tools.go

```diff
@@ -14,8 +16,6 @@
  * limitations under the License.
  */
 
-// +build tools
-
```

Comment:
why this change? what changed?

Reply:
as we agreed, I moved it before the copyright comment